packages/typo3-docs-theme/src/Settings/Typo3DocsThemeSettings.php

```diff
@@ -23,4 +23,12 @@ public function getSettings(string $key, string $default = ''): string
         }
         return $this->settings[$key];
     }
+
+    /**
+     * @return array<string, string>
+     */
+    public function getAllSettings(): array
```

Comment:
Can we have unit tests for this class (as you introduce a new method)?

Reply:
You want unit tests for the getters and setters on a data object? I find that quite superfluous

Reply:
You have logic in it (isset check). But I am fine to move that to a dedicated PR.

Reply:
Feel free to provide a PR